work.openpaper.Paperwork.json

```diff
@@ -0,0 +1,402 @@
+{
+    "app-id": "work.openpaper.Paperwork",
+    "branch": "stable",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.26",
+    "sdk": "org.gnome.Sdk",
+    "command": "paperwork",
+    "copy-icon": true,
+    "finish-args": [
+        "--share=ipc",
+        "--share=network",
+        "--socket=x11",
+        "--socket=wayland",
+        "--filesystem=home",
+        "--talk-name=org.freedesktop.Notifications",
+        "--talk-name=org.freedesktop.DBus",
+        "--socket=session-bus",
+        "--socket=system-bus",
```

Comment:
Does it need access to both session and system bus?

Reply:
Good point. The session bus is required to start Nautilus at some point, but I don't think Paperwork has any use for the system bus.